src/main/resources/i18n/strings.properties

```diff
@@ -318,9 +318,14 @@ preferences.volume.feature.readOnly=Read-only mount
 ## Updates
 preferences.updates=Updates
 preferences.updates.currentVersion=Current Version: %s
+preferences.updates.latestVersion=Latest Version: %s
 preferences.updates.autoUpdateCheck=Check for updates automatically
 preferences.updates.checkNowBtn=Check Now
 preferences.updates.updateAvailable=Update to version %s available.
+preferences.updates.lastUpdateCheck=The last update check was performed on: %s
```

Comment:
Do not make a whole sentence here, keep it like the rest: `Last check for updates:` or `Last update check:`